### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by redcarti**
PR of #413 

* [x] ``serv.locales.setStringLang(lang, path, value)``
* [x] ``serv.locales.setString(path, values)``
* [x] ``serv.locales.getString(lang, path)`` || ``serv.localeString(lang, path)``
* [x] ``player.localeString(path)``
* [x] local localization
  1. [x] localizations
  2. [x] add to ``serv.locales.langs``
* [x] docs
* [x] tests 

### Examples
Set string in one lang
```js
serv.locales.setStringLang('en_us', 'example.path', 'Hello!')
```

Set string to multiple langs
```js
serv.locales.setString('example.path.multiple', {
  en_US: 'Multiple languages',
  ru_RU: 'Несколько языков'
})
```

Get localized string
```js
serv.info(serv.localeString('en_US', 'example.path.multiple')) // return 'Multiple languages'
```

### Info
``serv.locales.setString`` replaces values.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "emit-then": "^2.0.0",
     "event-promise": "^0.0.1",
     "flatmap": "^0.0.3",
+    "lodash": "^4.17.15",
```
Comment (rom1504) [inline package.json:34]: I don't think we really need lodash for this

**Comment by rom1504** [comment]
can you rebase on master

**Comment by redcarti** [comment]
will rebase it later

File: package.json (hunk 1 above)
Comment (redcarti) [reply package.json:34]: I used lodash for get & set strings

**Comment by redcarti** [comment]
> where does the locale information come from ?

player locale? `settings` packet. or sth other?

File: package.json (hunk 1 above)
Comment (rom1504) [reply package.json:34]: yes, you can do it without using that lib, it's pretty big for this basic use case

**Comment by rom1504** [comment]
ah yeah right ok

File: package.json (hunk 1 above)
Comment (redcarti) [reply package.json:34]: hmm, well, will rewrite later

**Comment by rom1504** [comment]
the locale files should be in minecraft-data and can you PR any adaptation you made to the extractor in minecraft jar extractor ?

**Comment by redcarti** [comment]
> the locale files should be in minecraft-data and can you PR any adaptation you made to the extractor in minecraft jar extractor ?

I didn't use jar extractor, en-us lang stored only.

**Comment by rom1504** [comment]
> I didn't use jar extractor, en-us lang stored only.

ok could you adapt it and do it ?
doing this manually is quite annoying to maintain

**Comment by redcarti** [comment]
> ok could you adapt it and do it ?
> doing this manually is quite annoying to maintain

well I can create PR to mcdata and store langs there but it needs to be finded

**Comment by rom1504** [comment]
Do you think it would be very hard to adapt https://github.com/PrismarineJS/minecraft-jar-extractor/blob/master/lang.js to work for more languages ?

**Comment by redcarti** [comment]
I mean jar hasn't any langs, except en_us. So, as I said in #413, they stored locally in assets folder

**Comment by redcarti** [comment]
as I know.. but I checked jars and didn't found any langs

**Comment by rom1504** [comment]
asset folder can be downloaded using https://github.com/PrismarineJS/node-minecraft-wrap#launcherdownloadgetassetassetfileversion
see this example https://github.com/PrismarineJS/node-minecraft-wrap/blob/master/examples/exampleLauncherDownload.js

**Comment by redcarti** [comment]
oh well, okay. will try.

What about `localeBroadcast()`? I've tested it but the code is not that good...

**Comment by rom1504** [comment]
the languages stored should be in minecraft-data, not here https://github.com/PrismarineJS/minecraft-jar-extractor/pull/23#issuecomment-658254528

**Comment by rom1504** [comment]
@redcarti tests are failing ? any plan to finish this ?

**Comment by redcarti** [comment]
I wanted to log that langs are loaded and etc. but log.js plugin loaded after locale.js, so it kinda broked

**Comment by redcarti** [comment]
I think that log.js can be moved in index.js, where server is starting

**Comment by rom1504** [comment]
avoid to do things that plugin loading time and instead do it after an event is triggered for example

**Comment by redcarti** [comment]
okay, well

**Comment by redcarti** [comment]
I forgot about it, ehh.. So, it fails cuz I'm logging `Downloading langs...` & `Loading langs...` with `Loaded ${count} langs` and it says "Cannot log" or something like that

**Comment by redcarti** [comment]
Well, idk how to solve it...

**Pull request closed without merging**